### DISCUSSION TITLE
You exceeded your current quota, please check your plan and billing details.

### DISCUSSION THREAD

**Discussion by kobzar** [jackMort/ChatGPT.nvim#344] [Q&A] [answered]
This plugin only for prepaided GPT accounts?
I just install the plugin and add fresh key from my openAI account
But all tie have error
"You exceeded your current quota, please check your plan and billing details."

**Comment by kobzar** [comment]
i was check with both of keys with the same results.

**Comment by fegyi001** [comment]
Same here. Can use chat.openai.com without any notification about exceeding any quota, but cannot use this plugin at all.

**Comment by kobzar** [comment]
i try both variants with the same results

**Comment by marekzan** [comment]
chat.openai.com is free with the 3.5 model. If you want to use the API of OpenAI you need to buy credits.

**Comment by kobzar** [comment]
> chat.openai.com is free with the 3.5 model. If you want to use the API of OpenAI you need to buy credits.

You are don’t reading previous posts. I have two accounts, one of them is free and next one is prepared. Both of them return the same error.

**Comment by marekzan** [comment]
I have read all posts and you didn't mention that you tried it with an account where you have purchased credits.

**Comment by apojulsoph** [comment, marked as answer]
type `<C-n>` to edit a new session and clear the message if you use an account with credits

**Comment by YuCao16** [comment]
same here

**Comment by GitMurf** [comment]
I had the same problem and took a while to figure out that openai has changed the way they do their accounts and quota system from the last time I used the API! You now must put a minimum of $5 prepaid in your account and then turn on auto recharge to add $5 each time you get below your $5 threshold. Go here to set it up: https://platform.openai.com/account/billing/overview

![image](https://github.com/jackMort/ChatGPT.nvim/assets/64155612/33eaf817-b5d1-4787-815c-75e75494db0f)

cc @YuCao16 @kobzar @fegyi001
